azure/folder.go

```diff
@@ -93,14 +99,26 @@ func ConfigureFolder(prefix string, settings map[string]string) (storage.Folder,
 		return nil, NewFolderError(err, "Unable to create container")
 	}
 
+	var storageEndpointSuffix string
+	switch environmentName {
+	case "AzureUSGovernmentCloud":
+		storageEndpointSuffix = azure.USGovernmentCloud.StorageEndpointSuffix
+	case "AzureChinaCloud":
+		storageEndpointSuffix = azure.ChinaCloud.StorageEndpointSuffix
+	case "AzureGermanCloud":
+		storageEndpointSuffix = azure.GermanCloud.StorageEndpointSuffix
+	default:
+		storageEndpointSuffix = azure.PublicCloud.StorageEndpointSuffix
```

Comment:
Indent this maybe?

Reply:
Thank you for the comment  @na-jakobs

I am new in Golang and I not sure what is the recommended way to indent switch statements, but most examples I have seen online follow the same logic.

For example
https://github.com/golang/go/wiki/Switch